mysql-test/include/deadlock_stats.inc

```diff
@@ -0,0 +1,41 @@
+eval set $prior_engine_lwt = $engine_lwt;
+eval set $prior_engine_dld= $engine_dld;
+eval set global $raw_engine_lwt = 100000;
+eval set global $raw_engine_dld = ON;
+
+connect (con1,localhost,root,,);
```

Comment:
Since we're adding `--source include/count_sessions.inc` in the other tests, might as well add them here too.